hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -259,6 +263,10 @@ commitTx networkId scriptRegistry headId party commitBlueprintTx (initialInput,
       & bodyTxL . mintTxBodyL .~ mempty
       & addMetadata (mkHydraHeadV1TxName "CommitTx")
  where
+  addMetadata ::
+    TxMetadata ->
+    AlonzoTx StandardBabbage ->
+    AlonzoTx StandardBabbage
```

Comment:
```suggestion
    TxMetadata ->
    Ledger.Tx LedgerEra ->
    Ledger.Tx LedgerEra
```

If we really want to add this type signature, then we should be using types of already imported modules / packages we depend on.

This is related to the import comment above and applies to other signatures below too.